### PR TITLE
MODFQMMGR-555:fixing category translation

### PR DESCRIPTION
**Pull request by kjain110**

### CODE REVIEW

**Review by mweaver-ebsco — APPROVED** [review]

File: translations/mod-fqm-manager/en.json [hunk 1]
```diff
@@ -1082,8 +1082,8 @@
   "entityType.simple_organization.emails": "Emails",
   "entityType.simple_organization.emails.categories": "Category UUIDs",
   "entityType.simple_organization.emails.categories._qualified": "Emails category UUIDs",
-  "entityType.simple_organization.emails.categories_names": "Category names",
-  "entityType.simple_organization.emails.categories_names._qualified": "Emails category names",
+  "entityType.simple_organization.emails.categories_names": "Categories",
+  "entityType.simple_organization.emails.categories_names._qualified": "Emails categories",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:1086]:
```suggestion
  "entityType.simple_organization.emails.categories_names._qualified": "Email categories",
```

File: translations/mod-fqm-manager/en.json [hunk 2]
```diff
@@ -1114,8 +1114,8 @@
   "entityType.simple_organization.phone_numbers": "Phone numbers",
   "entityType.simple_organization.phone_numbers.categories": "Category UUIDs",
   "entityType.simple_organization.phone_numbers.categories._qualified": "Phone numbers category UUIDs",
-  "entityType.simple_organization.phone_numbers.categories_names": "Category Names",
-  "entityType.simple_organization.phone_numbers.categories_names._qualified": "Phone numbers category names",
+  "entityType.simple_organization.phone_numbers.categories_names": "Categories",
+  "entityType.simple_organization.phone_numbers.categories_names._qualified": "Phone numbers categories",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:1118]:
```suggestion
  "entityType.simple_organization.phone_numbers.categories_names._qualified": "Phone number categories",
```

**Comment by ncovercash** [comment]
Unless Kathleen specifically wants them with both plurals

File: translations/mod-fqm-manager/en.json [hunk 3]
```diff
@@ -1082,8 +1082,8 @@
   "entityType.simple_organization.emails": "Emails",
   "entityType.simple_organization.emails.categories": "Category UUIDs",
   "entityType.simple_organization.emails.categories._qualified": "Emails category UUIDs",
-  "entityType.simple_organization.emails.categories_names": "Category names",
-  "entityType.simple_organization.emails.categories_names._qualified": "Emails category names",
+  "entityType.simple_organization.emails.categories_names": "Categories",
+  "entityType.simple_organization.emails.categories_names._qualified": "Email categories",
   "entityType.simple_organization.emails.description": "Description",
   "entityType.simple_organization.emails.description._qualified": "Emails description",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:1088]: Would you mind fixing the others while you're already in there?

**Review by ncovercash — CHANGES_REQUESTED** [review]
see comment

**Comment by kjain110** [comment]
> see comment

asked Kathleen, we only need to category names to categories. No other changes needed as of @ncovercash

**Review by ncovercash — APPROVED** [review]

**Pull request merged**